action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
     description: 'Google Cloud region e.g. us-central1, us-east4'
     default: 'us-central1'
     required: false
+  add_comment:
+    description: 'Add a comment to an open PR as the final step - defaults to "true"'
+    default: 'true'
```

Comment:
Curious - why a string and not a boolean?

Reply:
Schema validation in VS Code yelled at me that it had to be a string. Looking at the docs the 'default' field on inputs only support string data. https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

Reply:
Lovely. Thanks